test/distributed/fsdp/test_fsdp_summon_full_params.py

```diff
@@ -7,26 +7,28 @@
 import torch
 import torch.nn as nn
 from torch import distributed as dist
-from torch.distributed.fsdp import CPUOffload, MixedPrecision
-from torch.distributed.fsdp import FlatParameter
+from torch.distributed.fsdp import CPUOffload
 from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
-from torch.distributed.fsdp.fully_sharded_data_parallel import FullyShardedDataParallel
-from torch.distributed.fsdp.wrap import wrap, enable_wrap
+from torch.distributed.fsdp import MixedPrecision
```

Comment:
any reason we separated the torch.distributed.fsdp imports?

Reply:
`FlatParamShardMetadata` and `FlattenParamsWrapper` are not exposed in `__init__.py`, so we have to import from the files themselves (`flat_param.py` and `flatten_params_wrapper.py`). It could be good to keep it like that since we do not want users to rely on those classes. `FlatParamShardMetadata` right now is only for testing, and `FlattenParamsWrapper` may be removed in the future (not sure).